sonar-plugin-api/src/main/java/org/sonar/api/database/DatabaseProperties.java

```diff
@@ -19,6 +19,10 @@
  */
 package org.sonar.api.database;
 
+/**
+ * @deprecated since 7.0 Should be moved out of the API
+ */
+@Deprecated
 public interface DatabaseProperties {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)